conf/upf.jsonc

```diff
@@ -155,7 +155,12 @@
     "cpiface": {
         "peers": ["148.162.12.214"],
         // [Optional] Below parameters
-        "dnn": "internet",
+        "dnn_list": [
+            {
+                "dnn": "internet",
+                "ue_ip_pool": "10.250.0.0/16"
```

Comment:
Given that now you have the `ue_ip_pool` here, should not it be removed from below?
```json
        // "hostname": "upf-0",
        "ue_ip_pool": "10.250.0.0/16"
    },
```